js-objects-and-arrays/js-objects-and-arrays_arrays/index.js

```diff
@@ -3,51 +3,56 @@ console.clear();
 // EXERCISE 1
 // Modify the array exampleArray so that it contains a number and a string.
 
-const exampleArray = [true];
+const exampleArray = [25, "hello"];
 
 // EXERCISE 2
 // Nest an array inside array nestedArray. After that nestedArray should contain an array as an element.
 
-const nestedArray = ["example", 10, true];
+const nestedArray = ["example", [25, "hello"], 10, true];
```

Comment:
Same comment here as above. Suggest to use maybe a push method to insert a new array element into the nestedArray object.